roles/goshimmer/tasks/deps_apt.yml

```diff
@@ -7,7 +7,7 @@
   apt_key:
     keyserver: keyserver.ubuntu.com
     id: 93C4A3FD7BB9C367
-  when: "[ansible_distribution] | intersect(['Debian', 'Raspbian']) | count"
+  when: "ansible_distribution == 'Debian' or ansible_lsb.id == 'Raspbian'"
```

Comment:
above intersect filter was used because, if you start getting more releases that should evaluate you get a long "or ... or ... or " so it makes it nicer to just append to the intersect list.